Networking/Networking/Remote/SystemPluginsRemote.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+
+/// SystemPlugins: Remote Endpoints
+///
+public class SystemPluginsRemote: Remote {
+
+    /// Retrieves all of the `SystemPlugin`s for a given site.
+    ///
+    /// - Parameters:
+    ///   - siteID: Site for which we'll fetch the system plugins.
+    ///   - completion: Closure to be executed upon completion.
+    ///
+    public func loadSystemPlugins(for siteID: Int64,
+                            completion: @escaping (Result<[SystemPlugin], Error>) -> Void) {
+        let path = Constants.systemPluginsPath
```

Comment:
Since the `system_status` resource contains a lot of information we don't need. I think it's worth to limit the endpoint to only return the "active_plugins" field.

You can take a look at how this is done here https://github.com/woocommerce/woocommerce-ios/blob/develop/Networking/Networking/Remote/ProductsRemote.swift#L232

Reply:
Wow, I didn't know that. Added!